.github/dependabot.yml

```diff
@@ -12,4 +12,4 @@ updates:
       time: "09:00"
       timezone: "America/New_York"
     # Allow up to 10 open pull requests for npm dependencies
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: 1
```

Comment:
I think we can use this feature in dependabot in this repo more easily so i'm going to leave it on and let it give us 1 PR a day until we get everything upgraded.